packages/liveblocks-yjs/package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "scripts": {
     "dev": "tsup --watch",
-    "build": "tsup && cp dist/index.d.ts dist/index.d.mts",
+    "build": "tsup",
```

Comment:
This was necessary in an older version of tsup, so we could simply remove this — sorry about the hassle, and thanks for catching! 👍

Reply:
Thanks for making it simpler!